src/component/registry/types.ts

```diff
@@ -376,6 +376,8 @@ export interface ShapeBpmnSemantic extends BaseBpmnSemantic {
   callActivityKind?: ShapeBpmnCallActivityKind;
   /** Set when the {@link BaseBpmnSemantic.kind} relates to a BPMN event. */
   eventDefinitionKind?: ShapeBpmnEventDefinitionKind;
+  linkEventSourceIds?: string[];
+  linkEventTargetId?: string;
```

Comment:
**nitpick:** properties are stored in alphabetical order. Please keep this order when adding new properties.

Reply:
Same answer: trying to group properties regarding their topic is subjective, while alphabetical order is just an easier convention to follow.

Reply:
done in 75308fd4
See also #2742

Reply:
done in 75308fd
See also #2742